ports/geckolib/glue.rs

```diff
@@ -454,10 +454,17 @@ pub extern "C" fn Servo_StyleSet_GetBaseComputedValuesForElement(raw_data: RawSe
         Some(PseudoElement::from_atom_unchecked(atom, /* anon_box = */ false))
     };
     let pseudos = &styles.pseudos;
-    let pseudo_style = pseudo.as_ref().map(|p| (p, pseudos.get(p).unwrap()));
+    let pseudo_style = match pseudo {
```

Comment:
Yeah, I could convert these into one-liners, but I usually prefer `.map` and `.and_then` when the closure doesn't capture any state from the outside, otherwise I think it just becomes a bit more difficult to read.

I think I still want to preserve the assertion, so I'd convert these to `pseudo.map(|p| pseudos.get(p).unwrap())`... I don't have a strong preference, wdyt?